swagger.json

```diff
@@ -1121,6 +1121,77 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos_daily_log": {
+            "patch": {
+              "consumes": [
+                "application/json"
+              ],
+              "description": "Modify details, such as trailer number or shipping ID, for a driver's daily log",
+              "operationId": "edit_fleet_drivers_hos_daily_log_details",
+              "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "description": "ID (integer) of the driver with HOS logs.",
+                  "in": "path",
+                  "name": "driver_id",
+                  "required": true,
+                  "type": "string"
+                },
+                {
+                  "in": "body",
+                  "name": "editHosDailyLogParams",
+                  "required": true,
+                  "schema": {
+                    "properties": {
+                      "timestamp": {
+                        "description": "A specified timestamp within the bounds of a driver's daily log, specified in milliseconds UNIX time.",
+                        "example": 1462881998034,
+                        "type": "integer"
+                      },
+                      "shippingId": {
+                        "description": "Shipping Id to add to driver's daily log.",
```

Comment:
Some more description here would be useful. It's not clear whether this is a samsara internal id or a customer specific id.